text/0000-innerhtml-binding.md

```diff
@@ -0,0 +1,108 @@
+- Start Date: 2019-10-03
+- RFC PR: (leave this empty)
+- Lightning Web Component Issue: (leave this empty)
+
+# Summary
+
+Applications dealing with rich content, like commerce apps, need to render 
+this content as raw HTML, typically coming from a database or a CMS. Today, 
+the data binding capability always escapes the HTML for security reasons,
+which makes the rich content data not rendered as expected.
+
+# Basic example
+
+The following template render some HTML content as innerHTML  
+
+```
+function createMarkup() { return {__html: 'First &middot; Second'}; };
+
+<template>
+    <div lwc:innerHTML={createMarkup()}/>

```

Comment:
HTML attribute names are case-insensitive and the parser will not be able to differentiate between `lwc:innerHTML` and `lwc:innerhtml`. I would rather the following directive names:
* `lwc:inner-html`
* `lwc:html`
* `lwc:raw`

Reply:
I would vote for the simple and meaningful `lwc:html` in that case. Also this syntax leaves us the opportunity to bolt on security/sanitization afterwards, similar to for instance Angular's DomSanitizer approach.

Reply:
I would like to keep LWC lightweight and not have to bring DOMPurify into LWC. I would rather delegate the sanitization to Locker if present, otherwise do a plain `Element.innerHTML`. @caridy what do you think?

Reply:
I agree with that